patch/0001-arm64-default-config-for-sonic-patches.patch

```diff
@@ -0,0 +1,74 @@
+From adf767005a35a2afeb8006c1023633af4de9d78e Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 17 Oct 2019 18:31:57 +0530
+Subject: [PATCH] arm64 default config for sonic patches
+
```

Comment:
Please elaborate. Currently it looks like, only `make olddefconfig` was run against a newer Linux kernel version. This could be split out into a separate merge/pull request, couldn’t it be?

Reply:
Yes, the patch approach for kernel config is obsolete in current master branch,

This PR has to be revisited and the required changes should go into patch/kconfig_inclusion and patch/kconfig_exclusion.
Thanks for your time to review this PR.  
Action Item would be to raise a as per new manage_kernel_config infra.